config.template.py

```diff
@@ -15,7 +15,18 @@
 # Caches followers to remove chat spam (Default: False)
 CACHE_FOLLOWS = False
 
-handlers = Handlers(LoggingHandler(), EventHandler(CACHE_FOLLOWS), 
+# How long in minutes before resending message, 0 = never resend
+# Only matters if CACHE_FOLLOWS = True (Default: 0)
+CACHE_FOLLOWS_TIME = 0
+
+
+# Don't touch below this line (The code bites)
+CACHE_DATA = {
```

Comment:
Rather than having the constants `CACHE_FOLLOWS` and `CACHE_FOLLOWS_TIME`, we should just use this dictionary.
```python
CACHE_DATA = {
    "CACHE_FOLLOWS": False,
    "CACHE_FOLLOWS_TIME": 0
}
```